ntbs-service/DataMigration/TreatmentEventMapper.cs

```diff
@@ -78,5 +80,31 @@ private async Task TryAddTbServiceAndCaseManagerToTreatmentEvent(TreatmentEvent
                 ev.CaseManagerId = (await _referenceDataRepository.GetUserByUsernameAsync(caseManagerUsername)).Id;
             }
         }
+
+        private string RemoveUnnecessaryNoteInfo(string note)
```

Comment:
My general feeling about detailed regexes like this, are that they're tricky to get right and then just as hard to debug. It's generally pretty difficult to account for all the possible edge cases. However, you've done the right thing here in making sure that we err on the side of returning too much information, so I'm happy enough to stick with it here.

Reply:
There are <10 comments in ETS that it is not matching, so where we will take the full comment.